vllm/distributed/eplb/eplb_state.py

```diff
@@ -244,7 +304,7 @@ def build(
             dtype=torch.int32,
             device=device,
         )
-        expert_load_window_size = parallel_config.eplb_config.window_size
+        expert_load_window_size = parallel_config.eplb_config.eplb_window_size
```

Comment:
Looks like this came from a bad merge (there are other places in this PR that this happens)

```suggestion
        expert_load_window_size = parallel_config.eplb_config.window_size
```